.github/workflows/build-gluon.yml

```diff
@@ -12,6 +12,10 @@ on:
 permissions:
   contents: read
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref || github.run_id }}-${{ github.event.pull_request.number || github.run_id }}
```

Comment:
This would also trigger on branches, which should not be interrupted. I'm fine with cancelling PR's though.

Reply:
For branches `github.event.pull_request.number` isn't set, therefore it falls back to `github.run_id` and that will be diffrent for diffent commits inside a branch.

Reply:
fair, i did not parse that correctly.